python/ray/experimental/workflow/storage/debug.py

```diff
@@ -148,5 +151,11 @@ def get_log(self, index: int) -> Any:
     def get_value(self, index: int, is_json: bool) -> Any:
         return self._logged_storage.get_value(index, is_json)
 
+    def log_off(self):
```

Comment:
are we using these/do we need them? 

i'm fine with keeping it if it's just helpful for debugging tbh.